railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%%= locale %>">
```

Comment:
I think those are attribute setters/getters on `I18n` though - have you tried just `default_locale` in a view?